lib/Utils/Backends.pm

```diff
@@ -161,4 +162,11 @@ sub is_pvm {
     return check_var('BACKEND', 'spvm') || check_var('BACKEND', 'pvm_hmc');
 }
 
+sub set_ssh_console_timeout {
+    my ($sshd_config_file, $sshd_timeout) = @_;
+    my $client_count_max = $sshd_timeout / 60;
```

Comment:
what about adding a comment for timeout unit? second or minute?

Reply:
Done